pkg/managerdriver/driver_test.go

```diff
@@ -290,6 +291,7 @@ var _ = Describe("Driver", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 
+			// TODO: Alice, fix broken
```

Comment:
Are these tests just incorrect because they relied on the old behavior which generated the tunnels no matter what? Or are they broken in some other way? If so, I don't mind fixing them since I was probably the one who broke them when I wrote them 😬

Reply:
Oops, I forgot to remove some of those comments 😓 
The tests are fixed now but were initially broken because they were expecting tunnels when there were no longer any tunnels (because the default mapping strategy on the ingresses creates endpoints). I fixed the test by adding the `mapping-strategy: edges` annotation and adding a new one to check for no tunnels when that annotation is not included, but I forgot to clean up the comments.